components/network/ntp/test/results-all.master

```diff
@@ -1,22 +1,25 @@
-crypto.c:14:test_MakeSHAKE128Mac:PASS
+make_mac: MAC SHAKE128 Digest Final failed.
```

Comment:
This is apparently a regression.  I've no idea what would be the impact of this but it could possibly cause that something won't work as expected.  Since `ntpd` is core part of system (and I would like to avoid reports that people are no longer able to sync their time properly) then please try to find why is this failing now.

Thank you.

Reply:
I allready  tried, but sorry I'm too dump to fix this, if you want to have that IPD4 part, you'll have to take this PR as it is otherwise tell me and I'll cancel it.

Reply:
> I allready tried, but sorry I'm too dump to fix this, if you want to have that IPD4 part, you'll have to take this PR as it is otherwise tell me and I'll cancel it.

Quick search revealed that the problem is very likely caused by a change in OpenSSL 3.4.

Related links:

- https://github.com/openssl/technical-policies/issues/93
- https://github.com/erlang/otp/issues/8997
- https://github.com/erlang/otp/pull/9002

Reply:
If you like just raise another PR and fix it by your own :-) - I have a wife ans other hobbies :-)